composer.json

```diff
@@ -13,8 +13,8 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "symfony/framework-bundle": "~2.1",
-        "doctrine/orm": ">=2.2.3,<2.4-dev",
+        "symfony/framework-bundle": ">=2.2.1,<2.7.*",
+        "doctrine/orm": ">=2.2.3,<2.5-dev",
```

Comment:
I've submitted a PR to set this to 2.4.* - I think there are quite a few breaking changes in 2.5 so we'll probably need to stick to 2.4